modules/local/validate_clinical/main.nf

```diff
@@ -0,0 +1,143 @@
+process VALIDATE_CLINICAL {
+    tag "$meta.id"
+    label 'process_single'
+    maxForks 1
+
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/ampcombi:2.0.1--pyhdfd78af_0':
+        'biocontainers/ampcombi:2.0.1--pyhdfd78af_0' }"
+
+    input:
+        //tuple val(meta), val(analysis), val(clinical), val(files), path(status_file), path(relational_mapping)
+        tuple val(meta), val(analysis), val(clinical), val(files), path(status_file), path(relational_mapping), path(analysis_types), path(data_directory)
+
+    output:
+        tuple val(meta), val(analysis), val(clinical), val(files), path("*_status.yml"), emit: status
```

Comment:
Be careful that the two `status.yml` files (input/output) could be both sent out to the output channel.